run.sh

```diff
@@ -12,13 +12,19 @@ run__dev () {
 	docker run --rm --interactive --tty --env-file .env.dev --volume ./src:/opt/root-pythia/src root-pythia-dev:latest
 }
 
+run__watch () {
+	# watch mode
+	docker build --file Dockerfile.watch -t root-pythia-dev:latest .
```

Comment:
shouldn't you call the image `root-pythia-watch:latest`?

Reply:
Yes it will be clearer indeed. Fixed also

Reply:
not really neither